src/pytorch_lightning/tuner/tuning.py

```diff
@@ -40,7 +40,7 @@ def _tune(
         scale_batch_size_kwargs = scale_batch_size_kwargs or {}
         lr_find_kwargs = lr_find_kwargs or {}
         # return a dict instead of a tuple so BC is not broken if a new tuning procedure is added
-        result = {}
+        result: Dict[str, Any] = {}
```

Comment:
```suggestion
        result: Dict[str, Union[int, _LRFinder]] = {}
```

Reply:
Thanks! I think the `Union` has to be optional otherwise `mypy` will throw an error: `Incompatible types in assignment (expression has type "Optional[int]", target has type "Union[int, _LRFinder]")`.
`

Reply:
```suggestion
        result: Dict[str, Optional[Union[int, _LRFinder]]] = {}
```